Resources/config/services/gateway.xml

```diff
@@ -32,5 +32,10 @@
         <service id="Sulu\Bundle\SyliusConsumerBundle\Gateway\ProductAttributeGatewayInterface" class="Sulu\Bundle\SyliusConsumerBundle\Gateway\ProductAttributeGateway">
             <argument type="service" id="sulu_sylius_consumer.gateway_client" />
         </service>
+
+        <service id="Sulu\Bundle\SyliusConsumerBundle\Gateway\ProductVariantChannelPricingGatewayInterface"
```

Comment:
same here you can let it as it is but its [bad practice](https://symfony.com/doc/current/bundles/best_practices.html#services) to us fcqn as ids in bundles.